src/test/nameservice.ts

```diff
@@ -641,4 +638,32 @@ describe('BlockstackService tests', () => {
     })
   })
 
+  describe("getCruxIDByAddress tests", () => {
+    it("126LEzWTg6twppHtJodwF8am8PwPdgbmwV using cruxdev wallet should resolve to ankit@cruxdev.crux", async () => {
+      const cruxIDs = await getCruxIDByAddress(config.BLOCKSTACK.BNS_NODES[0], "cruxdev", "126LEzWTg6twppHtJodwF8am8PwPdgbmwV")
```

Comment:
Can we add a case when more then one id is returned across different domains by API call?

Reply:
refactored the code to return only one cruxID.